SPECS/python-ffwd-client/python-ffwd-client.spec

```diff
@@ -0,0 +1,37 @@
+Name:           python-ffwd-client
+Version:        0.0.2
+Release:        1%{?dist}
+Summary:        Flexible metric forwarding agent(ffwd) client for python
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/udoprog/ffwd-client-python
+Source0:        https://github.com/udoprog/ffwd-client-python/archive/refs/tags/%{version}.tar.gz#/ffwd-client-python-%{version}.tar.gz
+BuildRequires:  python3-setuptools
+BuildRequires:  pyproject-rpm-macros
```

Comment:
I don't think you use any of the macros from this package

Reply:
removed this dependency.